gleam.toml

```diff
@@ -1,16 +1,9 @@
 name = "shork"
-version = "1.0.0"
+version = "0.1.0"
 
-# Fill out these fields if you intend to generate HTML documentation or publish
-# your project to the Hex package manager.
-#
-# description = ""
-# licences = ["Apache-2.0"]
-# repository = { type = "github", user = "", repo = "" }
-# links = [{ title = "Website", href = "" }]
-#
-# For a full reference of all the available options, you can have a look at
-# https://gleam.run/writing-gleam/gleam-toml/.
+description = "A MySQL / MariaDB database client for Gleam, based on MySQL-OTP. Heavily inspired by POG."
+licences = ["GPL-3.0"]
```

Comment:
I mean, it's your decision. But I think it might be worth thinking about using something like LGPL or even Apache2 or BSD for this. Otherwise you'd need to opensource all projects using this library.

Reply:
Hmm.. I used that license because [mysql-otp](https://github.com/mysql-otp/mysql-otp/blob/master/COPYING) uses it.

Reply:
Aaahh.. No, they are using LGPL.